### ISSUE TITLE
UUID lock contention

### ISSUE THREAD

**Issue by emolsson** [Ericsson/ecaudit#108]
While testing with a lot of concurrent batch operations it seems that generating [random UUIDs](https://github.com/Ericsson/ecaudit/blob/release/c3.0.11/ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/handler/AuditQueryHandler.java#L153) is causing a lot of contention. Under the hood random UUIDs uses SecureRandom which uses a synchronized code path (as well as occasionally using /dev/random or /dev/urandom).
On a node handling ~800 batches/s around 5% of the batch requests got blocked by this.

It would be good if we could postpone the UUID generation in some way or creating it in a way that avoids synchronization.
